addons/mail/static/src/discuss_app/discuss.xml

```diff
@@ -5,22 +5,36 @@
     <div class="o-mail-Discuss d-flex h-100 flex-grow-1 border-top" t-att-class="{ 'flex-column align-items-center bg-view': store.isSmall }" t-ref="root">
         <div t-if="store.isSmall and store.discuss.activeTab === 'mailbox'" class="w-100 border-bottom" t-call="mail.Discuss.mobileTopbar" t-ref="mobileTopbar"/>
         <div t-if="thread and !(store.isSmall and ['chat', 'channel'].includes(store.discuss.activeTab))" class="o-mail-Discuss-content d-flex flex-column h-100 w-100 bg-view overflow-auto" t-ref="content">
-            <div class="o-mail-Discuss-header px-3 bg-100 border-bottom d-flex flex-shrink-0" style="height:42px;">
-                <ThreadIcon className="'me-2 align-self-center'" thread="thread"/>
+            <div class="o-mail-Discuss-header px-3 bg-view border-bottom d-flex flex-shrink-0 align-items-center" style="height:46px;">
+                <div t-if="thread and thread.model === 'discuss.channel'" class="o-mail-Discuss-threadAvatar position-relative align-self-center align-items-center mt-2 mb-2 me-2">
+                    <img class="rounded" t-att-src="channelAvatar" alt="Thread Image"/>
+                    <FileUploader t-if="thread.is_editable and thread.type !== 'chat'" acceptedFileExtensions="'.bmp, .jpg, .jpeg, .png, .svg'" showUploadingText="false" multiUpload="false" onUploaded="(data) => this.onFileUploaded(data)">
```

Comment:
just `thread.is_editable`

Reply:
No, otherwise you are allowed to edit avatar of DM, which shouldn't be the case

Reply:
this was together with the other comment, I understand we don't want to change the avatar of a specific person.

Reply:
Using just `is_editable` makes sense for me too (see other comment)